docs/HowTo/High-Availability.md

```diff
@@ -8,7 +8,12 @@ By placing the instances behind a load balancer, downtime can be limited during
 
 ![Orion HA](../images/Orion-HA.png)
 
-# Storage for highly available scenarios
+## Orion keys
+
+When configuring for high availability, configure the nodes in the Orion cluster (Orion A and Orion B in the
+above diagram) with the same set of keys and advertise the load balancer address.
```

Comment:
I think the part about advertising the lb address should be added in the Load Balancer section.

Reply:
Fixed